apps/nextjs/src/lib/errors/server/middlewareErrorLogging.ts

```diff
@@ -0,0 +1,71 @@
+import * as Sentry from "@sentry/nextjs";
```

Comment:
It potentially feels like quite a disconnected place to put this file, and I wonder if it should sit alongside the middleware?

Reply:
Happy to move it!